Practice/Practice/Views/Employee/Index.cshtml

```diff
@@ -59,7 +59,11 @@
                 <td>
                     @Html.ActionLink("Edit", "Edit", new {  id=item.Id }) |
                     @Html.ActionLink("Details", "Details", new { /* id=item.PrimaryKey */ }) |
-                    @Html.ActionLink("Delete", "Delete", new { /* id=item.PrimaryKey */ })
+                  
+                   @using(Html.BeginForm("Delete", "Employee", new { id = item.Id }))
```

Comment:
Nie ma sensu tworzyć form skoro można wywołać określoną akcję z przekazaniem parametru. Jedyne co należy jeszcze zrobić to wyświetlać modal przechodni, na którym użytkownik będzie mógł potwierdzić, że chce usunąć konkretnego pracownika

Reply:
Dzięki.
Poprawiłem i zrobiłem pull request.